### ISSUE TITLE
GDPR - not working

### ISSUE THREAD

**Issue by Luxx99** [YetiForceCompany/YetiForceCRM#8754]
Hi,
I'm setting up YetiForce 4.4 and I'm dealing with a issue in the GDPR module.

Assuming that all the needed modules are installed properly, as shown in the picture below,

![gdpr installed modules](https://user-images.githubusercontent.com/23698008/49385536-f4ebb380-f71d-11e8-89da-1881ab76aad1.jpg)

I click on one menu item, like "DataSet Register". I see a nearly blank scrren, without a table - with columns names - to list the items.

![gdpr items list](https://user-images.githubusercontent.com/23698008/49385606-1b115380-f71e-11e8-99f5-b19b987f1a41.jpg)

If I click on "Create Record", I get a blank page like below.

![gdpr item creation](https://user-images.githubusercontent.com/23698008/49385790-7f341780-f71e-11e8-9b70-98f9f7db21e8.jpg)

This behaviour is the same for all menu items in the GDPR.

Please help. Is this a wrong configuration of something?
thank you in advance
regards
Fabrizio

**Comment by Luxx99** [comment]
-Issue: I described in details the error in the post above with screenshots. Basically I cannot create any of the items related to the GDPR. 
-How to trigger the error: I attach a GIF
![gdpr](https://user-images.githubusercontent.com/23698008/49400539-db119700-f744-11e8-98d3-3152438ec739.gif)
-My configuration
![yetiforce_settings](https://user-images.githubusercontent.com/23698008/49400564-ea90e000-f744-11e8-8af9-e3ceff372070.png)
-Logs: there are no relevant logs to attach, even if the debug_mode is ON
-Environment: YF 4.4
-Browser: Firefox Quantum 63.0.3 (64 bit) and Chrome 73.0.3629.0 (Build ufficiale) canary (32 bit)
-server linux hosted by siteground

**Comment by mariuszkrzaczkowski** [comment]
without logs we are not able to help you.
try to show the error on the gitdev version

**Comment by Luxx99** [comment]
I changed the configuration of server, so that now respect all parameters, just to avoid problems due to misconfiguration.
![yetiforce_settings 1](https://user-images.githubusercontent.com/23698008/49426742-b992c880-f7a1-11e8-8a13-e40524036ebe.png)
Besides I forced YF to generate the log file below. What inside was generated just doing the following action:
1. click on menu item "Data Set Register"
2. click on "Create New Record"
3. click on "cancel"
[system.log](https://github.com/YetiForceCompany/YetiForceCRM/files/2642872/system.log)
